src/cluster_legacy.c

```diff
@@ -6593,25 +6593,27 @@ int clusterCommandSpecial(client *c) {
         }
         resetManualFailover();
         server.cluster->mf_end = mstime() + CLUSTER_MF_TIMEOUT;
+        sds client = catClientInfoString(sdsempty(), c);
```

Comment:
One slight concern I have with this is that this will log "user data". I wasn't able to find the corresponding Redis PR/Issue, but we considered the "Name" to be user supplied data, and we didn't want to put it in the log for any compliance reasons. I'm fine with other information getting logged though, like IP and port.

Reply:
make sense. though we have a lot of logs in networking.c that call client info, such as output buffer limit / client evict, do we has plan to remove it too?

the link is https://github.com/redis/redis/pull/11747/. Do we want to pick it up again? i see Redis merged a new PR based on this

Reply:
Yeah, restarted the conversation here: https://github.com/valkey-io/valkey/pull/877. I think we can modify catClientInfoString to just omit name and user based off of the flag being proposed.